test/Test.cpp

```diff
@@ -91,6 +91,19 @@ TEST(trace, Array)
   mbed_tracef(TRACE_LEVEL_DEBUG, "mygr", "%s", mbed_trace_array(longStr, 200) );
 }
 
+TEST(trace, Null0Array)
+{
+  static const unsigned char array[2] = { 0x23, 0x45 };
+  mbed_tracef(TRACE_LEVEL_DEBUG, "mygr", "%s", mbed_trace_array(array, 2));
```

Comment:
So, is the correct function to use now mbed_trace_array? or tr_array() or trace_array()? I have now seen 3 different versions.